examples/dt/neutron/3cont_2comp_1ipa-ipv4-vlan-dvr

```diff
@@ -0,0 +1,63 @@
+# Deployed Topology 3cont_2comp_1ipa-ipv4-vlan-dvr-ovn
+
+## General information
+
+| Revision | Change              |  Date      |
+| -------: | :------------------ | :-----:    |
+|     v0.1 | Initial publication | 2024-01-10 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, VLAN                | Must have          |
+| Glance    | default                      | Must have          |
+| Cinder    | default                      | Must have          |
+| Keystone  | default                      | Must have          |
+| TLS       | default                      | Must have          |
+| Compute   | DifferentHostFilter          | Must have          |
+
+### Additional information 
+This DT uses FreeIPA
+
+#### Nova
+
+#### Cinder
+
+#### OVN
+add configuration of ipv4
+--network-force-compute-dvr yes
```

Comment:
IMO, we can omit this since the job has dvr enabled. Formerly (before 17.1) this job was with centralized routers and we nevertheless wanted compute nodes to be connected via br-ex to external network. Now this is a redundant setting.

Reply:
Fixed